src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -1,8 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-linux`
Test-Connection.Ping.Default parameter set is 'Ping'
```powershell
Expected the value to have type [System.Net.NetworkInformation.PingOptions] or any of its subtypes, but got $null with type $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 41
41:             $result.Options | Should -BeOfType "System.Net.NetworkInformation.PingOptions"
```
Test-Connection.Ping.Force IPv4 with explicit PingOptions
```powershell
Expected $true, but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 99
99:                 $result1[0].Options.DontFragment | Should -BeTrue
```